cluster-provision/gocli/cmd/provision_manager.go

```diff
@@ -314,21 +314,36 @@ func getKubevirtciTag() (string, error) {
 }
 
 func runCommand(command string, args []string) (string, error) {
-	var stdout, stderr bytes.Buffer
+	var stderr1 bytes.Buffer
+	remoteAddCmd := exec.Command("git", "remote", "add", "upstream", "https://github.com/kubevirt/kubevirtci.git")
+	remoteAddCmd.Stderr = &stderr1
+	if err := remoteAddCmd.Run(); err != nil {
+		logrus.Debugf("Failed to add remote named upstream: %v, stderr: %s", err, stderr1.String()) //upstream remote already may exist
+	} else {
+		logrus.Debug("Successfully added remote named upstream.")
+	}
+
+	var stderr2 bytes.Buffer
```

Comment:
can we not just reuse sdtout and stderr for each of these rather than stderr3, stderr2 etc?